### PR TITLE
[ADD] support more search operators when search for relations

### PR DESCRIPTION
**Pull request by hbrunn**
This allows for queries like 'Has relation of type [contains] $substring_or_relation_name'

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by nbessi** [comment]
:+1:

File: partner_relations/i18n/nl.po [hunk 1]
```diff
@@ -247,6 +247,12 @@ msgstr "De begindatum mag niet na de einddatum liggen."
 #. module: partner_relations
 #: code:addons/partner_relations/model/res_partner_relation.py:260
 #, python-format
+msgid "Unsupported search operand \"%s\""
+msgstr "Operator \"%s\" is niet ondersteund"
```
Comment (pedrobaeza) [inline partner_relations/i18n/nl.po:251]: And what about the .pot file?

Comment (hbrunn) [inline partner_relations/i18n/nl.po:251]: right, the pot file. Thanks!

**Pull request merged**